proj.sh

```diff
@@ -54,6 +54,7 @@ function deploy-stack {
   docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml --project-name metabolicatlas up --detach --build --force-recreate --remove-orphans --renew-anon-volumes
   docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml exec -d pg psql -U postgres -c "alter user postgres with password '${POSTGRES_PASSWORD}';"
   CHOSEN_ENV=$METATLAS_DEFAULT_ENV
+  export DOCKER_CONTEXT=default
```

Comment:
```sh
function deploy-stack {
  trap "unset DOCKER_CONTEXT" RETURN
  CHOSEN_ENV="env-${1:-$LOCALENV}.env"
  generate-data
  docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml --project-name metabolicatlas up --detach --build --force-recreate --remove-orphans --renew-anon-volumes
  docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml exec -d pg psql -U postgres -c "alter user postgres with password '${POSTGRES_PASSWORD}';"
  CHOSEN_ENV=$METATLAS_DEFAULT_ENV
}
```
This works for me, for handling cases when you have killed the deploy with ctrl+c. Would be nice if someone else could test it!

Reply:
Thanks @MalinAhlberg! I was thinking about this before but I'm not sure if this captures all types of interruptions, for example `ctrl+d` and `ctrl+z`. Will try to give this a try now.

Reply:
I tried this and had to add a variant for `zsh`:

Please give this a try now when you have time. I can verify that it (resetting context) works running `Ctrl + c` but does not work with `Ctrl + \` for me. It is still better than not having any at all though. Thanks for mentioning this @mihai-sysbio and to @MalinAhlberg again for the suggested solution.

Reply:
Nice @e0!

For other reviewers: I found a problem with this solution on Ubuntu, am looking into it now.

Reply:
This seems to work for both me and @e0  (see [slack thread](https://nbisweden.slack.com/archives/C02NW26HZ/p1667384187045479) for more info).
```
function deploy-stack {
  trap 'unset DOCKER_CONTEXT; trap - INT' INT TERM
  CHOSEN_ENV="env-${1:-$LOCALENV}.env"
  generate-data
  docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml --project-name metabolicatlas up --detach --build --force-recreate --remove-orphans --renew-anon-volumes
  docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml exec -d pg psql -U postgres -c "alter user postgres with password '${POSTGRES_PASSWORD}';"
  CHOSEN_ENV=$METATLAS_DEFAULT_ENV
  export DOCKER_CONTEXT=default
}
```